Frontend/Admin/src/app/components/station-view/list-stations/station-management/station-management.component.ts

```diff
@@ -2,7 +2,7 @@ import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
 import {BikeStation} from '../../../../models/bikeStation';
 import {StationService} from '../../../../services/station.service';
 import {NotificationService} from '../../../../services/notification.service';
-import { BikeStationExtended } from 'src/app/models/bikeStationWithDetails';
+import { BikeStationExtended } from 'src/app/models/BIkeServiceExtended';
```

Comment:
tu jest BI jakoś dziwnie wielkimi literami, ale nie nalegam na zmianę, możne to literówka albo specjalnie

Reply:
Literówka. Już poprawiam

Reply:
Dobra, nie poprawiam bo już zmergowane xD